code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -504,7 +504,7 @@ char *translate_key(char *key)
 	// both key and joystick button are mapped to this control
 	if ((key_code >= 0 ) && (joy_code >= 0) ) {
 		strcpy_s(text, key_text);
-		strcat_s(text, " or ");
+		strcat_s(text, XSTR( " or ", 1638));
```

Comment:
I think the white space should not be part of the translatable string since that could lead to some easy mistakes:
```suggestion
		strcat_s(text, " ");
		strcat_s(text, XSTR("or", 1638));
		strcat_s(text, " ");
```

Reply:
That make sense, yes.